Dockerfile

```
@@ -1,14 +1,8 @@
-FROM python:3.11.4-buster
-
+FROM python:3.11.4-alpine
 ENV REVIEWDOG_VERSION=v0.17.0
-
-RUN set -eux \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends \
-        git \
-        wget \
+RUN apk --no-cache add git gcc musl-dev \
```

Was gcc and musl-dev only added for the docker push workflow? If so, then let's make sure to remove these, please.

```suggestion
RUN apk --no-cache add git \
```